plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplitSource.java

```diff
@@ -157,7 +156,6 @@ public IcebergSplitSource(
         this.fieldIdToType = primitiveFieldTypes(tableScan.schema());
         this.dynamicFilter = requireNonNull(dynamicFilter, "dynamicFilter is null");
         this.dynamicFilteringWaitTimeoutMillis = dynamicFilteringWaitTimeout.toMillis();
-        this.executor = executor;
```

Comment:
should `Remove unused variable` commit be squashed?

Reply:
I'll squash all the commits into one